crates/cli-support/src/js/mod.rs

```diff
@@ -476,7 +485,10 @@ impl<'a> Context<'a> {
                 );
 
                 if needs_manual_start {
-                    start = Some("\nwasm.__wbindgen_start();\n".to_string());
+                    _start.push_str("\nwasm.__wbindgen_start();\n");
+                }
+                if !_start.is_empty() {
+                    start = Some(_start);
                 }
```

Comment:
I'm assuming this logic is a refactoring leftover?
I assume this doesn't have to happen before `self.imports_post.push_str()`, so it could all be at the bottom, not requiring the `_start` variable.

Let me know if I'm missing something.